src/main/resources/defaults/displayViews.yml

```diff
@@ -1122,6 +1122,10 @@
           template: "defaults/displayViews/documents/overview/abstract.html"
           field: abstract
           order: 1
+        - name: name of conference
+          template: "defaults/displayViews/documents/overview/nameOfConference.html"
+          field: nameOfConference
+          order: 4
```

Comment:
This has order of 4, is this intended?

If so, then please consider moving it next to `publication outlet` for numerical consistency.
If not, then the orders need to be changed.